src/test/java/api/support/fixtures/LoansFixture.java

```diff
@@ -154,6 +156,16 @@ public Response declareItemLost(JsonObject loanJson) {
     return declareItemLost(loanId, builder);
   }
 
+  public Response changeDueDate(ChangeDueDateRequestBuilder request) {
+    return restAssuredClient.post(request.create(),
+      changeDueDateURL(request.getLoanId()), 204, "change-due-date-request");
+  }
+
+  public Response attemptChangeDueDate(ChangeDueDateRequestBuilder request) {
+    return restAssuredClient.post(request.create(),
+        changeDueDateURL(request.getLoanId()), "change-due-date-request");
+  }
+
```

Comment:
I think this fixture is getting pretty big and it could be better to split it up into separate fixtures for different parts of the loan lifecycle e.g. CheckOutFixture, ChangeDueDateFixture etc